app/locales/en/messages.po

```diff
@@ -59,7 +59,7 @@ msgstr "Categories"
 
 #: app/configurator/components/dataset-preview.tsx:60
 msgid "browse.dataset.create-visualization"
-msgstr "Create visualization"
+msgstr "Start a visualization"
```

Comment:
I guess other languages should be change right ?

"Démarrer une visualisation" would be the French translation

Reply:
Saw the other PRs.